index.js

```diff
@@ -93,15 +93,24 @@ function handleShell(fn, cmd, opts) {
 	return fn(file, args, opts);
 }
 
-function getStream(stream, encoding) {
-	if (!stream) {
+function getStream(process, stream, encoding, maxBuffer) {
+	if (!process[stream]) {
 		return null;
 	}
 
+	var p;
+
 	if (encoding) {
-		return _getStream(stream, {encoding: encoding});
+		p = _getStream(process[stream], {encoding: encoding, maxBuffer: maxBuffer});
+	} else {
+		p = _getStream.buffer(process[stream], {maxBuffer: maxBuffer});
 	}
-	return _getStream.buffer(stream);
+
+	return p.catch(function (e) {
+		e.stream = stream;
```

Comment:
Why isn't https://github.com/sindresorhus/eslint-plugin-xo/blob/master/docs/rules/catch-error-name.md catching this?